docs/clearml_sdk/task_sdk.md

```diff
@@ -671,7 +671,7 @@ task.connect(params_dictionary)
 
 ### Configuration Objects
 
-Configuration objects more elaborate than a key-value dictionary (such as nested dictionaries or configuration files), 
+Configuration objects are more elaborate than a key-value dictionary (such as nested dictionaries or configuration files), 
```

Comment:
@pollfly  "Configuration objects are more elaborate than a key-value dictionary (such as nested dictionaries or configuration files), can be logged to a task using the" -> "To log configuration more elaborate than a key-value dictionary (such as nested dictionaries or configuration files) use the"